private/rdf4cpp/writer/WriteQuad.hpp

```diff
@@ -39,6 +39,11 @@ bool write_pred(Node const pred, writer::BufWriterParts const writer) {
         return false;                       \
     }
 
+[[nodiscard]] inline bool is_default_graph(Node const &graph) noexcept {
+    auto const g = graph.as_iri();
+    return !g.null() && g.is_default_graph();
```

Comment:
why check that it is not the null node?

Reply:
Good point, might as well directly call the function

Reply:
patched